hieradata_aws/class/training/db_admin.yaml

```diff
@@ -43,3 +43,201 @@ govuk_env_sync::tasks:
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_content_audit_tool_training_daily":
+    ensure: "present"
+    hour: "3"
+    minute: "30"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "content_audit_tool_production"
+    temppath: "/tmp/content_audit_tool_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_content_publisher_training_daily":
+    ensure: "present"
+    hour: "3"
+    minute: "30"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "content_publisher_production"
+    temppath: "/tmp/content_publisher_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_content_tagger_training_daily":
+    ensure: "present"
+    hour: "4"
+    minute: "30"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "content_tagger_production"
+    temppath: "/tmp/content_tagger_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_email_alert_api_training_daily":
+    ensure: "present"
+    hour: "2"
+    minute: "20"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "email-alert-api_production"
+    temppath: "/tmp/email-alert-api_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_email_alert_monitor_training_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "10"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "email-alert-monitor_production"
+    temppath: "/tmp/email-alert-monitor_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_organisations_publisher_training_daily":
+    ensure: "present"
+    hour: "2"
+    minute: "10"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "organisations-publisher_production"
+    temppath: "/tmp/organisations-publisher_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_organisations_publisher_training_daily":
+    ensure: "present"
+    hour: "3"
+    minute: "30"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "organisations-publisher_production"
+    temppath: "/tmp/organisations-publisher_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_service_manual_publisher_training_daily":
+    ensure: "present"
+    hour: "3"
+    minute: "35"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "service-manual-publisher_production"
+    temppath: "/tmp/service-manual-publisher_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_publishing_api_training_daily":
+    ensure: "present"
+    hour: "3"
```

Comment:
This is a big one - will likely take more than 1h to complete - maybe go for 5:00, here...